components/button/test/button.perf.js

```diff
@@ -0,0 +1,24 @@
+import '../button.js';
+import { html, testRenderTime } from 'web-test-runner-performance/browser.js';
+import { expect } from '@open-wc/testing';
+
+describe('d2l-button', () => {
+
+	it('normal', async() => {
+		const element = html`<d2l-button>Button</d2l-button>`;
+		const result = await testRenderTime(element, { iterations: 1 });
```

Comment:
1 iteration likely isn't a good enough sample? Looks like the default is 3 though... so should we just always stick with that? I see you've used 1, 100, 500 and 1000... we should probably pick one and stick to it unless there's a good reason to deviate.

Reply:
Think I fat fingered this one when I was switching iteration counts